.github/workflows/lint.yaml

```diff
@@ -20,4 +20,4 @@ jobs:
           # Fail if "new" violations detected or "any", default "new"
           failIf: new
           # Additional arguments to pass to flake8, default "." (current directory)
-          args: "--max-line-length=99 --exclude=lib/installed_clients/,lib/AbstractHandle/AbstractHandleServer.py --ignore=E305,E265 ."
+          args: "--max-line-length=99 --exclude=lib/installed_clients/,lib/AbstractHandle/AbstractHandleServer.py --ignore=E305,E265,E501 ."
```

Comment:
This seems like it prevents linting these errors for all files, which seems wrong. Presumably you just want an exclude list for generated files

Reply:
fixed 👍

Reply:
@Tianhao-Gu is there a reason we're globally ignoring E305 and E265?

Reply:
I cannot recall any reasons we are ignoring those warnings. If we remove `E305,E265` from ignore, which file fails?

Reply:
E265 block comment should start with '# ', mostly in impl files. 
E305 expected 2 blank lines after class or function definition, found 1, just in log.py file

Reply:
yea I guess if you don't want to ignore globally just exclude those files.

Reply:
You don't want to ignore impl, that has non gen code

Reply:
I would probably just fix log.py and remove 305 in that case, and leave 265 there

Reply:
I think we should ignore E265 because of generated files.

Reply:
👍

Reply:
Sorry, i flipped 265 and 305. Ignore 305, fix log.py, and remove 265

Reply:
> ignore E265

I meant keep it in the cli line